RoomsQuickstart/Program.cs

```diff
@@ -95,7 +121,42 @@ static async Task UpdateRoom(string roomId)
             }
         }
 
-        static public async Task AddParticipants(string roomId, string[]?roomParticipants = null)
+        static async Task GetRoom(string roomId)
+        {
+            try
+            {
+                Console.WriteLine("\n---------Get Room---------\n");
+                CancellationToken cancellationToken = new CancellationTokenSource().Token;
+
+                CommunicationRoom updatedRoom = await RoomCollection.GetRoomAsync(roomId, cancellationToken);
+                PrintRoom(updatedRoom);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Failed to get room, id: {roomId}, ex --> {ex}");
+            }
+        }
+
+        static async Task ListRoom()
+        {
+            try
+            {
+                Console.WriteLine("\n---------List Rooms---------\n");
+                CancellationToken cancellationToken = new CancellationTokenSource().Token;
+
+                AsyncPageable<CommunicationRoom> allRooms = RoomCollection.GetRoomsAsync(cancellationToken);
+                await foreach (CommunicationRoom room in allRooms)
```

Comment:
similarly, we may say to list only the first 1 room to avoid going through a whole collection

Reply:
Addressed, thanks for the suggestion